### DISCUSSION TITLE
How to use Create App

### DISCUSSION THREAD

**Discussion by jtxxviii** [netless-io/flat#1848] [General]
Greetings Agora developers, 

I would like to know how to use this '' Create App ". Is it possible that I can add youtube or a web browser using Create App? 
![Create app](https://user-images.githubusercontent.com/126577964/222125176-b468e45d-e2cf-4ef5-b697-c4b7464ed52b.PNG)

Thank you.

**Comment by hyrious** [comment]
@BlackHole1

**Comment by BlackHole1** [comment]
@jtxxviii This is a developer-oriented feature, similar to Twitter supports registration/login via google.
If you want to add to your site: Register/Login via Agora Flat. Then this feature is exactly what you need.

**Reply by jtxxviii** [reply]
@BlackHole1 Sir, where can we get authorization callback url? Thanks.

**Reply by BlackHole1** [reply]
`authorization callback url` is your server API.

**Reply by BlackHole1** [reply]
https://oauth.net/2/
